### ISSUE TITLE
"banishes" detected as NOUN rather than VERB

### ISSUE THREAD

**Issue by bboyle** [winkjs/wink-eng-lite-web-model#9]
The POS tagger is detecting "banishes" as a noun instead of a verb. Not sure if this is a gap in the english model, some other bug, or something I've coded wrong. Here's the test case I put together:

```js
import winkNLP from 'wink-nlp';
import model from 'wink-eng-lite-web-model';

describe('winkNLP', () => {
  const nlp = winkNLP(model);
  const its = nlp.its;
    
  it('detects banishes as a verb', () => {
    const text = 'Prince Escalus banishes Romeo.';
    const doc = nlp.readDoc(text);
    const tokens = doc.tokens();
    expect(tokens.out(its.pos)).toEqual(['PROPN', 'PROPN', 'VERB', 'PROPN', 'PUNCT']);
  });
});
```

![image](https://user-images.githubusercontent.com/559114/213953260-d33416a7-0404-49a5-8bbb-c060b563335a.png)

**Comment by sanjayaksaxena** [comment]
Hello @bboyle 

Thanks for highlighting the issue. The POS tagger accuracy, like any other tagger, is not 100% and therefore there is always a likelihood of a few incorrect tags.

We will surely look into the issue, but it may take a bit longer as it involves retraining & revalidation.

We would also request you to consider using the [web model](https://github.com/winkjs/wink-eng-lite-web-model) instead of this one as we will not be supporting this in future.

Best,
Sanjaya

**Comment by bboyle** [comment]
Thanks Sanjaya. I am using the web model, but apologies for opening the issue on the wrong repo. Interested to hear what you may find out with retraining — I found several similar verbs that were not detected which was pretty confusing. Wink is amazing otherwise :)

**Comment by sanjayaksaxena** [comment]
Thank you @bboyle for you contribution; we will initiate the work soon and keep you updated here. Moving this issue to the web model's repo.

**Comment by rachnachakraborty** [comment]
Hello @bboyle 

We have initiated work on the web model to address incorrect tagging of verbs.

Shall keep you posted.

Thanks,
Rachna

**Comment by sanjayaksaxena** [comment]
Hello @bboyle 

Have released the updated model (version 1.5.0) on NPM.

Best,
Sanjaya
